Makefile

```diff
@@ -199,7 +199,15 @@ _push_logger_image:
 
 logger: deps _build_logger_image _push_logger_image
 
-build: deps vdbench fio iometer mysql-client tpcc-client mongo-client jenkins-client postgres-client custom-client libiscsi logger 
+build: deps vdbench fio iometer mysql-client tpcc-client mongo-client jenkins-client postgres-client custom-client libiscsi logger
```

Comment:
also build target has to come after you define the steps for gitlab-runner & also include the gitlab-runner dep